src/main/java/team/asd/service/CalendarService.java

```diff
@@ -0,0 +1,108 @@
+package team.asd.service;
+
+import team.asd.constants.DateElement;
+import team.asd.exceptions.WrongArgumentException;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.WeekFields;
+import java.util.Locale;
+import java.util.Objects;
+
+public class CalendarService implements IsCalendarService {
+	final private DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+
+	@Override
+	public String toString(LocalDate date) {
+		if (Objects.isNull(date)) {
+			return null;
+		}
+
+		return date.format(dateFormatter);
+	}
+
+	@Override
+	public LocalDate toLocalDate(String stringDate) {
+		if (Objects.isNull(stringDate)) {
+			return null;
+		}
+
+		try {
+			return LocalDate.parse(stringDate, dateFormatter);
+		} catch (Exception e) {
+			return null;
+		}
+	}
+
+	@Override
+	public long defineCountInRange(LocalDate fromDate, LocalDate toDate, ChronoUnit unit) {
+		if (Objects.isNull(unit) || Objects.isNull(toDate) || Objects.isNull(fromDate)) {
+			throw new WrongArgumentException("Parameter is wrong");
+		}
+
+		if (unit.equals(ChronoUnit.SECONDS)) {
```

Comment:
What if I will add more tests with `MILLIS`, `MINUTES`, `HOURS`?

Reply:
And... it failed 😅
https://github.com/demurelebowski/asd-java-intern/actions/runs/3327419489/jobs/5502207315